docs/src/markdown/usage.md

```diff
@@ -368,16 +368,16 @@ When `selection_only` is enabled, you might have a regular expression chain that
     },
 ```
 
-## Apply Regular Expressions Right Before File Save Event
+## Apply Regular Expressions Right before File Save Event
```

Comment:
Before should probably remain title cased here.

Reply:
Depends which style sytem you follow. I passed the titles through the Chicago style (CMS), which converts "before" to lowercase — MLA style too. The APA and AP styles would have it as "Before":

https://capitalizemytitle.com/

As a long as consistent styling is applied, anyone of them should be ok. I went for Chicago style because it's not too academic.